pkg/agents/team7/agent4/agent.go

```diff
@@ -0,0 +1,62 @@
+package team7agent4
+
+import (
+	"github.com/SOMAS2021/SOMAS2021/pkg/infra"
+)
+
+type CustomAgent1 struct {
```

Comment:
```suggestion
type CustomAgent4 struct {
```
This should be changed for all instances.